thirdparty/download-thirdparty.sh

```diff
@@ -291,12 +291,12 @@ echo "Finished patching $LZ4_SOURCE"
 
 # brpc patch to disable shared library
 cd $TP_SOURCE_DIR/$BRPC_SOURCE
-if [ ! -f $PATCHED_MARK ] && [ $BRPC_SOURCE == "incubator-brpc-0.9.5" ]; then
-    patch -p1 < $TP_PATCH_DIR/incubator-brpc-0.9.5.patch
+if [ ! -f $PATCHED_MARK ] && [ $BRPC_SOURCE == "brpc-0.9.5" ]; then
```

Comment:
we should delete obsoleted patches since brpc version upgraded to 1.3.0

Reply:
I agree with you, but I think there should be another special pr to deal with it. Other libraries also have this problem. For example, the current version of glog is 0.4, but the code and patches of 0.3 also exist